jmxtrans-output/jmxtrans-output-elastic/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.jmxtrans</groupId>
+		<artifactId>jmxtrans-output</artifactId>
+		<version>251-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>jmxtrans-output-elastic</artifactId>
+
+	<name>JmxTrans - ElasticSearch output writers</name>
+
+	<properties>
+		<verify.mutationThreshold>80</verify.mutationThreshold>
```

Comment:
I have managed to get to 60 percent. Interesting concept, this mutation testing. It is new to me. I have been struggling to get to higher percentage. Lowered the minimum to 60.